src/lib/string_sign/tests/test_string_sign.ml

```diff
@@ -191,6 +191,46 @@ let test_secret_key_between_scalar_field_and_base_field () =
         "Sign and verify with secret key in scalar field" true
         (verify ~signature_kind signature keypair.public_key s) )
 
+let test_regression_signature () =
+  let inputs =
+    [ ( Signature_lib.Private_key.of_string_exn
+          "28948022309329048855892746252171976963363056481941560715954676764349967630337"
+      , Mina_signature_kind_type.Mainnet )
+    ; ( Signature_lib.Private_key.of_string_exn
+          "28948022309329048855892746252171976963363056481941560715954676764349967630337"
+      , Mina_signature_kind_type.Testnet )
+    ]
+  in
+  let exp_output =
+    [ ( "10098659636052751402960513659673534058318534694837361110423199873688986365908"
+      , "7844530585816769124362208605281765856859093320029640751494040544682079710450"
+      )
+    ; ( "19528641019288828403170135634909672400537532147682562327700497341129840775560"
+      , "24640829174932073706335857230587954505385529622139330517748682166416541732513"
+      )
+    ]
+  in
+  let l = List.zip_exn inputs exp_output in
+  List.iter l ~f:(fun ((sk, network), (exp_r_str, exp_s_str)) ->
+      let secret_key = sk in
+      let signature_kind = network in
```

Comment:
Nit: these alias are not necessary if we name these var correctly in function params.